game.html

```diff
@@ -2,21 +2,57 @@
 
 (function() {
 
-  var truth = 13;
+  var truth = 4;
+  var checkCorrect = function(answer) {
+    if(answer === truth) {
+      alert('Indeed, the path to enlightenment comes from exploring all possible paths ahead of you while leaving the past where it is.');
+      return true;
+    } else {
+      return false;
+    }
+  }
```

Comment:
Always end an anonymous function with a semi-colon (in this case, line 13 of your code). Line 13 would look like this:

}; //end anonymous function of var checkCorrect